src/app/admin/students/students.component.ts

```diff
@@ -7,9 +16,52 @@ import { Component, OnInit } from '@angular/core';
 })
 export class StudentsComponent implements OnInit {
 
-  constructor() { }
+  groupdID: number;
+
+  public STUDENTS_LIST: Array<GetStudentsInterface> = [];
```

Comment:
Uppercase with snakecase can be used for constants.
Shorthand GetStudentsInterface[] is preferable. FYI: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#caveats

Reply:
Got it, should it change it now or can I change it in the next pull request with ADD and UPDATE?

Reply:
Can be next request